ocs_ci/deployment/hosted_cluster.py

```diff
@@ -199,23 +196,53 @@ def config_has_hosted_odf_image(self, cluster_name):
 
         return regestry_exists and version_exists
 
-    def deploy_hosted_ocp_clusters(
-        self,
-    ):
+    def storage_installation_requested(self, cluster_name):
+        """
+        Check if the storage client installation was requested in the config
+
+        Args:
+            cluster_name: str: Name of the cluster
```

Comment:
You can add Returns section to the docstring

Reply:
fixed